src/app/view/components/login/dot-login-component/login-container.component.ts

```diff
@@ -1,13 +1,8 @@
 import { Component, ViewEncapsulation } from '@angular/core';
```

Comment:
I don't know if we should keep this class, when we start programing in angular we thought that create a container for some components was a good idea but really we are not doing it anymore 

the same for:

- https://github.com/dotCMS/core-web/blob/master/src/app/view/components/login/reset-password-component/reset-password-container.component.ts

- https://github.com/dotCMS/core-web/blob/master/src/app/view/components/login/forgot-password-component/forgot-password-container.component.ts